scipy/optimize/_linprog_highs.py

```diff
@@ -330,5 +345,10 @@ def _linprog_highs(lp, solver, time_limit=None, presolve=True,
            'message': statuses[res['status']][1],
            'nit': res.get('simplex_nit', 0) or res.get('ipm_nit', 0),
            'crossover_nit': res.get('crossover_nit'),
+           'method': solvers[solver],
+           'simplex_dual_edge_weight_strategy':
+               (simplex_dual_edge_weight_strategy if solver != "ipm"
+                else None)
```

Comment:
Can `_highs_wrapper` return the edge weight strategy used?

Reply:
No, for the same reason as above.

Reply:
OK, thanks.

Reply:
If @mckib2 agrees I'll just remove this, i guess.

Reply:
@mdhaber Yes, it can be removed